common/src/java/org/apache/hadoop/hive/ql/ErrorMsg.java

```diff
@@ -478,7 +478,7 @@ public enum ErrorMsg {
   DATACONNECTOR_NOT_EXISTS(10428, "Dataconnector does not exist:"),
   TIME_TRAVEL_NOT_ALLOWED(10429, "Time travel is not allowed for {0}. Please choose a storage format which supports the feature.", true),
   INVALID_METADATA_TABLE_NAME(10430, "Invalid metadata table name {0}.", true),
-  METADATA_TABLE_NOT_SUPPORTED(10431, "Metadata tables are not supported for table {0}.", true),
+  TABLE_IDENTIFIER_NOT_SUPPORTED(10431, "Table identifier is not supported for table {0}.", true),
```

Comment:
That looks like a different errors, isn't it?

Reply:
It essentially means the same thing that this table doesn't support three layer namespace, e.g ` dbName.tblName.metaTable`  or  `dbName.tblName.branch_branchName`.

The previos errors just only considered the iceberg meta table ` dbName.tblName.metaTable`.  here i refined it and I want to use a general term `Table identifier` to say that this table supports three layer namespace

Reply:
maybe `Table Metadata Ref extension not supported for table`

Reply:
I like this. It's better than identifier. Done.